src/library_gl.js

```diff
@@ -3817,6 +3856,30 @@ var LibraryGL = {
 #endif
   },
 
+#if GL_STATE_CACHE
```

Comment:
These one exist to route the default names for these functions to other names, which can then be referenced from `gl_cache.c`, which contain the injected versions. So applications first call the GL functions in `gl_cache.c`, which then call these renamed functions.